### DISCUSSION TITLE
Display as "You did ...." when I changed something in the room instead of the generic "MXID did ...."

### DISCUSSION THREAD

**Discussion by HarHarLinks** [element-hq/element-meta#149] [Ideas]
When I add a widget, android says "You added .... widget". Desktop says "... widget added by Displayname". This also applies for many/all other room configuration changes like that.

These should be unified across platforms, and on top I think the "You" method is nicer from a user perspective.

**Comment by daniellekirkwood** [comment]
This is a great enhancement to suggest, much more user friendly. 

Are there other examples of this, or is it the Widget case?

**Reply by HarHarLinks** [reply]
Inviting users, updating room name, -avatar, -topic, changing ACLs, pretty much everything.

**Reply by daniellekirkwood** [reply]
Thank you!

**Reply by HarHarLinks** [reply]
Setting your own name or avatar

**Comment by HarHarLinks** [comment]
also related https://github.com/vector-im/element-web/issues/3969 i.e. there are currently inconsistencies whether displayname or mxid are used which is pretty bad.

I just came across the example, quoting element web:
> Bob changed the power level of @bob:example.org from Admin to Default

as an example for multiple issues:
- Different ways (displayname, mxid) to refer to the same person. Imo it should both just use pills instead.
- Since it's the 2 person twice, it should instead say "Bob changed their own power level" if possible
- I'm not sure an admin should be able to demote themselves, but that is technically unrelated
